### PR TITLE
wifi_add minor usability tweaks

### PR DESCRIPTION
**Pull request by karlp**

### CODE REVIEW

File: modules/luci-mod-admin-full/luasrc/model/cbi/admin_network/wifi_add.lua [hunk 1]
```diff
@@ -16,7 +16,7 @@ if not iw then
 	return
 end
 
-m = SimpleForm("network", translate("Join Network: Settings"))
+m = SimpleForm("network", translate("Joining Network: ") .. http.formvalue("join"))
```
Comment (jow-) [inline modules/luci-mod-admin-full/luasrc/model/cbi/admin_network/wifi_add.lua]: Can you use a placeholder here? This makes it easier to translate into other languages, e.g.
`translate("Joining Network %q", http.formvalue("join"))`

Comment (karlp) [inline modules/luci-mod-admin-full/luasrc/model/cbi/admin_network/wifi_add.lua]: sure, will do

**Comment by karlp** [comment]
Changes requested have been made, is there anything else on this one?

**Comment by jow-** [comment]
Nope, LGTM now.

**Comment by augustgermar** [comment]
Hi guys. Great work! Just one minor change please, have a look when you have time:
https://github.com/openwrt/luci/pull/876

**Pull request merged**